rh-shield-operator/Dockerfile

```diff
@@ -0,0 +1,7 @@
+# Build the manager binary
+FROM quay.io/operator-framework/helm-operator:v1.36.1
```

Comment:
Not strictly related, but we can check to have dependabot configured to update this as a new image is out?

Reply:
Yes, this will be a part of follow up steps to build out the CI components. 👍🏻